src/generation-logic/generate.js

```diff
@@ -51,14 +54,19 @@ function getRandom(n, data) {
     return arr[0];
 }
 
+function getRandomRoundedPercentage(step) {
+    let rnd = Math.random();
+    return PERCENTAGE_MIN + Math.floor(rnd * (PERCENTAGE_MAX + step - PERCENTAGE_MIN) / step) * step;
+}
+
 function getQuarantineDecision() {
     let rnd = Math.random();
     if (rnd <= 0.25) {
         return "zwolnione z kwarantanny";
     }
     else {
         rnd = Math.random();
-        const quarantine_days = QUARANTINE_MIN + Math.floor(rnd * (QUARANTINE_MAX - QUARANTINE_MIN));
+        const quarantine_days = QUARANTINE_MIN + Math.floor(rnd * (QUARANTINE_MAX + 1 - QUARANTINE_MIN));
```

Comment:
I think we could simply use `Math.round()` instead, this should make reaching the top value much more probable

Reply:
While it's true, it would also make lower bound less probable. Generally using `round` would cause top and bottom values to appear about half as often as other values between. Using `floor` with `+ 1` gives all numbers in the range the same probability (which you may or may not want).